### PR TITLE
Improved output_path behaviour.

### PR DESCRIPTION
**Pull request by jcampbell05**
Cert now takes a leave out of Gym's book and creates the output folder if it doesn't exist.

Cert also expands relative paths so it should work from fastlane or the command line. Previously relative paths only worked from the command line.

### CODE REVIEW

File: spec/runner_spec.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
       Cert::Runner.new.launch
       expect(ENV["CER_CERTIFICATE_ID"]).to eq("cert_id")
-      expect(ENV["CER_FILE_PATH"]).to eq("./cert_id.cer")
+      expect(ENV["CER_FILE_PATH"]).to eq("#{Dir.pwd}/cert_id.cer")
```
Comment (KrauseFx) [inline spec/runner_spec.rb]: Why did you change that test?

**Comment by KrauseFx** [comment]
Looks great, thanks. I added a comment :+1:

File: spec/runner_spec.rb (hunk 1 above)
Comment (jcampbell05) [inline spec/runner_spec.rb]: In order for cert to work with both relative and absolute paths, I had to use File.expand to convert the relative path to an absolute (The same way Gym does).

This test fails unless I change it to be the absolute path. 

I have just noticed a weird quirk though with my solution, that when ran under fast lane if you put "./cache" it actually creates a folder called "Cache" in the fastfile folder, is this correct ?

**Comment by jcampbell05** [comment]
@KrauseFx  Added one back, if you could take a look.

**Comment by KrauseFx** [comment]
Looks great, thanks @jcampbell05 :+1:
